src/playlist-loader.js

```diff
@@ -207,9 +207,11 @@ const PlaylistLoader = function(srcUrl, withCredentials) {
 
   loader.stopRequest = () => {
     if (request) {
-      request.onreadystatechange = null;
-      request.abort();
+      let oldRequest = request;
```

Comment:
I couldn't think of a better name. I was thinking that "r" was too ambiguous, but I suppose it is better than "oldRequest." Any suggestions though?